suites/quincy/baremetal_pipeline/mero_4_node_rados_db_wal_conf.yaml

```diff
@@ -92,6 +92,66 @@ tests:
       destroy-cluster: false
       module: test_client.py
       name: "configure client"
+  -
+    test:
+      abort-on-fail: true
+      config:
+        command: add
+        copy_admin_keyring: true
+        id: client.1
+        install_packages:
+          - ceph-common
+          - cephadm
```

Comment:
we can remove the cephadm package from client.

Reply:
Removed the cephadm package